### ISSUE TITLE
Rust version does not build at all

### ISSUE THREAD

**Issue by LudwikJaniuk** [futurice/space-tyckiting#12]
`cargo build` just gives loads of errors.

**Comment by seance** [comment]
The Rust client is outdated. Any fixes will be appreciated!

**Comment by LudwikJaniuk** [comment]
Yeah, in the end we just rebuilt everything from scratch. But if you're going to use this project again, we can probably contribute an older version of our code after the finals are over (with all the AI stripped out, of course).

**Comment by seance** [comment]
@LudwikJaniuk looking forward to that PR we discussed! :)

**Comment by LudwikJaniuk** [comment]
I'll look at it around wednesday but it's going up, for sure :)

**Comment by phadej** [comment]
cc @daniellandau
